recipes/cfm-id/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set CONDA_BOOST = "1.66" %}
+package:
+  name: cfm-id
+  version: "33"
+
+source:
+  # the download is a mirror of a snapshot of 
+  # https://sourceforge.net/code-snapshots/svn/c/cf/cfm-id/code/cfm-id-code-r33-cfm.zip
+  # which is only present for 24h or so
+  url: https://depot.galaxyproject.org/software/cfm/cfm_r33_src_all.zip
+  md5: f02c79016eae6ad95413213509513eed
+  patches:
+    - CMakeLists.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - boost
+    - lp_solve
+    - rdkit ==2018.09.1
+  run:
+    - boost
+    - lp_solve
+    - rdkit ==2018.09.1
+
+# files taken from https://sourceforge.net/p/cfm-id/code/HEAD/tree/supplementary_material/cfm-train_example/train_example/
+test:
+  source_files:
```

Comment:
Yep, but the only way to run the program is to run it with data. There is no `--help` or `--version` option. So I do not see another way .. except to inject it into the zip at cargo port which I do not like since I like the idea that its identical to the sourceforge version.

Reply:
I'm actually surprised that this passes the tests. Since in the mulled test with Docker these files are not available. You could download the test files in the build.sh script with wget and put them in the package, but them finding them from the commandline might be tricky.  Your call.

Reply:
I'm surprised that they are supposed to fail. For curiosity I will change the file names in the command line and see if this fails.

Reply:
OK. So if I use wrong file names in the test then I get a segmentation fault .. a clear indication that the files are actually available and used in the test.